packaging/RPMS/Fedora/rabbitmq-server.spec

```diff
@@ -188,7 +190,9 @@ fi
 %attr(0755, rabbitmq, rabbitmq) %dir %{_localstatedir}/lib/rabbitmq
 %attr(0750, rabbitmq, rabbitmq) %dir %{_localstatedir}/lib/rabbitmq/mnesia
 %attr(0755, rabbitmq, rabbitmq) %dir %{_localstatedir}/log/rabbitmq
-%dir %{_sysconfdir}/rabbitmq
+%attr(2755, -, -) %dir %{_sysconfdir}/rabbitmq
```

Comment:
It works with both, `2750` is a bit more restrictive so more secure. In general we should not have side effects, since we are expanding the permissions to the `rabbitmq` group.